### ISSUE TITLE
ClassNotFoundException is occurred when forgetme.sh is run

### ISSUE THREAD

**Issue by chaminda** [wso2/product-sp#490]
**Description:**
Getting below exception when user runs the forgetme.sh

```java
2018-03-07 10:49:42 INFO  ForgetMeTool:167 - Generating pseudonym as pseudo name is not provided : d4abfc52-3a9c-436b-9eb2-95d7b598f06f
Exception in thread "main" java.util.ServiceConfigurationError: org.wso2.carbon.privacy.forgetme.api.runtime.InstructionReader: Provider org.wso2.carbon.privacy.forgetme.analytics.streams.instructions.AnalyticsStreamsInstructionReader could not be instantiated
	at java.util.ServiceLoader.fail(ServiceLoader.java:232)
	at java.util.ServiceLoader.access$100(ServiceLoader.java:185)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:384)
	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404)
	at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
	at java.lang.Iterable.forEach(Iterable.java:74)
	at org.wso2.carbon.privacy.forgetme.ConfigReader.<init>(ConfigReader.java:76)
	at org.wso2.carbon.privacy.forgetme.ConfigReader.<clinit>(ConfigReader.java:63)
	at org.wso2.carbon.privacy.forgetme.ForgetMeTool.process(ForgetMeTool.java:182)
	at org.wso2.carbon.privacy.forgetme.ForgetMeTool.main(ForgetMeTool.java:128)
Caused by: java.lang.NoClassDefFoundError: com/google/gson/JsonParseException
	at java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
	at java.lang.Class.getConstructor0(Class.java:3075)
	at java.lang.Class.newInstance(Class.java:412)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:380)
	... 7 more
Caused by: java.lang.ClassNotFoundException: com.google.gson.JsonParseException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:338)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 12 more
```

**Suggested Labels:**
Priority/Highest, Type/Bug

**Affected Product Version:**
wso2sp-4.0.0-update8

**OS, DB, other environment details and versions:**    
H2, macOS, jdk1.8

**Steps to reproduce:**
1. Start all the profiles; editor, dashboard, manager, worker.
2. Shutdown all the server instances
3. run ./bin/forgetme.sh

**Comment by this** [comment]
@pcnfernando did we fixed this for wso2sp-4.0.0-update9 release?

**Comment by pcnfernando** [comment]
This is resolved in wso2sp-4.0.0-update9 release.

**Comment by chaminda** [comment]
Verified on RC2
